Octokit/Models/Response/Verification.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Diagnostics;
+using System.Globalization;
+using Octokit.Internal;
+
+namespace Octokit.Models.Response
```

Comment:
There's a coding convention here that I guess needs to be documented properly. Despite these files living in `Octokit/Models/Responses` the corresponding namespace for these is just `Octokit`. This means new `using` statements are unnecessary for consumers (and for this PR).